app/routes/tutorials/$tutorialSlug/index.tsx

```diff
@@ -35,13 +35,21 @@ export default function Index() {
   return (
     <main className="container mx-auto p-4">
       <div className="prose prose-amber max-w-6xl bg-white mx-auto p-8 rounded mt-16 ">
-        <h1>Blog: {tutorial.fields.title}</h1>
+        <h1>{tutorial.fields.title}</h1>
         <div
           className="mt-4"
           dangerouslySetInnerHTML={{
             __html: sanitizeHtml(marked(tutorial.fields.content || '')),
           }}
         />
+        {tutorial.fields.lessons && tutorial.fields.lessons.length > 0 ? (
+          <Link
+            to={`lessons/${tutorial.fields.lessons[0].fields.slug}`}
+            className="group relative flex w-full justify-center rounded-md border border-transparent bg-amber-600 py-2 px-4 text-sm font-medium text-white hover:bg-amber-700 focus:outline-none focus:ring-2 focus:ring-amber-500 focus:ring-offset-2"
+          >
+            Start Learning
+          </Link>
+        ) : null}
```

Comment:
You're returning all the tutorial object from your loader, just to read its title, content and slug.
How many more fields are present in the tutorial ?

Depending on your answer, you could return just the `title`, `content` and `slug` variables from your loader, and keep a JSX clean (no ifs)